docs/topics/testing/tools.txt

```diff
@@ -880,6 +880,33 @@ It also provides an additional method:
     modify them, you could reload them in the ``setUp()`` method with
     :meth:`~django.db.models.Model.refresh_from_db`, for example.
 
+.. note::
+
+    Testing code which uses :func:`~django.db.transaction.on_commit` hooks are
```

Comment:
Maybe we can simplify doc string a little. I don't believe it will help may people to into the details of why transactions are not committed. Besides, that is explained somewhere else.

I would also try to avoid passive language, maybe something like this:

```
Testing code which uses :func:`~django.db.transaction.on_commit` hooks are
not run during the execution of :class:`~django.tests.TestCase` tests. As mentioned above, the proper way
to test this is to use the
:class:`~django.tests.TransactionTestCase`. However, should it be required to trigger
:func:`~django.db.transaction.on_commit` in a
:class:`~django.tests.TestCase` tests, one possible solution is to add a context manager as follows::
```